gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -354,6 +348,7 @@ gst_tensor_src_iio_channel_properties_free (gpointer data)
   g_free (prop->generic_name);
   g_free (prop->base_dir);
   g_free (prop->base_file);
+  g_free (prop);
```

Comment:
plz check the function gst_tensor_channel_list_filter_enabled()
```
    gst_tensor_src_iio_channel_properties_free (channel);
    *list_addr = g_list_remove (list, channel);
```

Reply:
Good catch, done.